src/spi/Adafruit_FlashTransport_SPI.cpp

```diff
@@ -108,11 +108,9 @@ void Adafruit_FlashTransport_SPI::fillAddress(uint8_t *buf, uint32_t addr) {
   switch (_addr_len) {
   case 4:
     *buf++ = (addr >> 24) & 0xFF;
-    __attribute__((fallthrough));
```

Comment:
removing this will cause gcc emit fall through warning when compiling with all warning option. please add these back.

Reply:
Added `__attribute__((fallthrough));` back in.